Microsoft.Research/Contracts/MsCorlib/System.Threading.Tasks.Task.cs

```diff
@@ -288,14 +294,26 @@ public static TaskFactory Factory
     //
     // Returns:
     //     true if the task has completed due to being canceled; otherwise false.
-    //public bool IsCanceled { get; }
+    public bool IsCanceled
```

Comment:
Potentially we can add a postcondition that the result of the property is true, when the status is `Cancelled`.